### DISCUSSION TITLE
OMR Bypass questions

### DISCUSSION THREAD

**Discussion by serlowenbrau** [Ysurac/openmptcprouter#2660] [Q&A]
I thought I had a decent understanding of omr-bypass, but now I am not so sure.  My wife wanted to try starz out, but I can't get around a vpn message for them.  It is not in the services list, so I can't use that.  I have tried bypassing starz.com and asn AS39984 (and just 39984, not sure of the syntax here).  No luck.  I am probably messing myself up by using my amplifi HD router behind the omr router to handle all the wifi and schedule controls, as I can't use IP or mac address to bypass that way.  I could put the amplifi in bridge mode, but then I lose all the easy profile controls to stop my kids from trying to stay up all night playing on the internet.  I'm sure I could probably accomplish the same thing on the omr router, but I don't know how currently.  Am I missing something simple?

**Comment by serlowenbrau** [comment]
I found 2 peer ASNs for starz and put those in, too.  Now it works.  :)

**Reply by SpectroHA** [reply]
Can you share what those are?
